fcs/docsrc/tools/generate.fsx

```diff
@@ -3,6 +3,9 @@
 // (the generated documentation is stored in the 'docs' directory)
```

Comment:
basically any change in here can be ignored for the moment, as docgen overall is broken on .net core.